test/Constraints/bridging.swift

```diff
@@ -241,9 +241,11 @@ func rdar19770981(_ s: String, ns: NSString) {
   f(ns) // expected-error{{'NSString' is not implicitly convertible to 'String'; did you mean to use 'as' to explicitly convert?}}{{7-7= as String}}
   f(ns as String)
   // 'as' has higher precedence than '>' so no parens are necessary with the fixit:
-  s > ns // expected-error{{'NSString' is not implicitly convertible to 'String'; did you mean to use 'as' to explicitly convert?}}{{9-9= as String}}
+  s > ns // expected-error{{binary operator '>' cannot be applied to operands of type 'String' and 'NSString'}}
```

Comment:
I was planning on opening a bug to investigate this. I am not sure these are improvements - I think the others were specifically intended for this case.